### ISSUE TITLE
Same addresses for different versions of contracts

### ISSUE THREAD

**Issue by nksazonov** [zerodevapp/kernel#88]
Good afternoon!
We are using Kernel SW and validators extensively, and are happy with updates frequency (v2.0, v2.1...).
The latest (v2.4) release has updated `ECDSAValidator` and `SessionKeyValidator`, however, their addresses (specified at the bottom of the `README.md`) were left unchanged.

Could you specify the addresses of updated validator modules in the readme, please?

**Comment by leekt** [comment]
for ECDSAValidator and SessionKeyValidator, the code should have not been changed but pushed the wrong code by mistake.
So the address of the ECDSA validator and session key validator is not changed actually

**Comment by nksazonov** [comment]
Thanks for the response!

---

> ... the code should have not been changed but pushed the wrong code by mistake

Do you mean the code is not production-ready yet?

**Comment by leekt** [comment]
Actually, it is production-ready, sorry if there was a confusion

Status for the each of the validators
- SessionKeyValidator has been used with `0xB8E3c4bEaACAd06f6092793012DA4a8cB23D6123` since last november and you should be using this if you are using kernel 2.x, so we made this to show up in all versions to not make things confusing

- ECDSAValidator this has been left unchanged, so you can use the version identified on the readme.md

the reason why i said " the code should have not been changed but pushed the wrong code by mistake." is that some commits that i tried while building v2.4 has change the interface in IKernelValidator, but i reverted it back to what it used to be before i published v2.4. So if you check the code, it has not been changed at all(for ECDSAValidator)

So tl;dr, the addresses shown on the readme is the latest address
